test/v1/fixtures/ListTransactionFixture.scala

```diff
@@ -18,44 +18,44 @@ package v1.fixtures
 
 import play.api.libs.json.{JsValue, Json}
 import uk.gov.hmrc.domain.Nino
-import v1.models.request.retrieveTransactions.RetrieveTransactionsParsedRequest
-import v1.models.response.retrieveTransaction.{RetrieveTransactionsResponse, TransactionItem}
+import v1.models.request.listTransactions.ListTransactionsParsedRequest
+import v1.models.response.listTransaction.{RetrieveTransactionsResponse, TransactionItem}
 
-object RetrieveTransactionFixture {
+object ListTransactionFixture {
```

Comment:
the documentId field needs to be included in all desJson vals, and the transactionId field needs to included in all mtdJson vals and response models.